docs/Governance.md

```diff
@@ -115,6 +115,17 @@ Responsibilities:
 * Manage github settings such as branch protection rules and repository secrets.
 * Resolve any disputes among maintainers.
 
+## Prioritizing Issues
```

Comment:
Since the "Reviewing Pull Requests" section is not modified, I assume there's no changes proposed to how prioritization is done at merge time, correct?  (If correct, that's fine with me.)

Reply:
Correct.